CIbuild.sh

```diff
@@ -6,6 +6,9 @@ export MCSERVER_BUILD_SERIES_NAME="Travis $CC $TRAVIS_MCSERVER_BUILD_TYPE"
 export MCSERVER_BUILD_ID=$TRAVIS_JOB_NUMBER
 export MCSERVER_BUILD_DATETIME=`date`
 
+if [ "$CXX" == "g++" ]; then
+	export CXX="/usr/bin/g++-4.8"
+fi
```

Comment:
This could use a comment saying why you're overriding the CXX variable. Also, shouldn't CC be overridden as well?